src/mantarray_desktop_app/server.py

```diff
@@ -972,10 +973,26 @@ def after_request(response: Response) -> Response:
         response = Response(status="404 Route not implemented")
     elif "get_available_data" in rule.rule and response.status_code == 200:
         del response_json["waveform_data"]["basic_data"]
+    elif "system_status" in rule.rule:
+        mantarray_nicknames = response_json.get("mantarray_nickname", {})
+        for board in mantarray_nicknames:
+            mantarray_nicknames[board] = "*" * len(mantarray_nicknames[board])
+    elif "set_mantarray_nickname" in rule.rule:
+        response_json["mantarray_nickname"] = "*" * len(
```

Comment:
same not as above about a function to convert a sensitive item to the way we display it when redacted